.php-cs-fixer.php-highest.php

```diff
@@ -10,16 +10,16 @@
  * with this source code in the file LICENSE.
  */
 
-if (PHP_VERSION_ID <= 80001 || PHP_VERSION_ID >= 80100) {
-    fwrite(STDERR, "PHP CS Fixer's config for PHP-HIGHEST can be executed only on highest supported PHP version - 8.0.*.\n");
+if (PHP_VERSION_ID <= 80001 || PHP_VERSION_ID >= 80200) {
```

Comment:
```suggestion
if (PHP_VERSION_ID < 80100 || PHP_VERSION_ID >= 80200) {
```

Reply:
I think the issue on 8.0.0 was solved in 8.0.1, so we can run on anything above it

Reply:
this check was to ensure that `php-highest` is executed only on ~8.0.1 originally, and now we should ensure it's only ~8.1.
you allow ~8.0 as well